ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -136,3 +143,31 @@ message PartyDetails {
   // Required
   bool is_local = 3;
 }
+
+message AddPartyPropertiesRequest {
+  // Required
+  string party = 1;
+  // The properties to add.
+  // Ignores adding duplicate properties.
+  // If you need to update a property, you need to delete it first using a separate call.
+  // Required
+  repeated Property properties = 2;
+}
+message AddPartyPropertiesResponse {
+  // The properties that were newly added by the request.
+  // Required
+  repeated Property newly_added_properties = 1;
```

Comment:
Response could just simply transfer back party details containing the properties

Reply:
FYI
Changed the design to update properties with 'replace-all' semantics using the UpdateUser/Party call, so for the moment this discussion is not applicable.